src/main/java/duke/Exception/DukeException.java

```diff
@@ -0,0 +1,14 @@
+package duke.Exception;
+
+public class DukeException extends Exception {
```

Comment:
Perhaps there could have been more exception classes to handle different types of errors? Your TypeOfException is a good example for the Duke specific formatting erorrs.